spinnaker_wrappers/CameraWrapper.h

```diff
@@ -25,6 +25,7 @@ class CameraWrapper {
     CameraWrapper(const CameraWrapper& wrapper);
     virtual ~CameraWrapper();
     virtual void Init();
+    virtual bool IsInitialized();
```

Comment:
not sure why these aren't const.
i guess i'd stick with their scheme, but curious why it is set-up this way.